app/index.html

```diff
@@ -387,13 +387,80 @@
       <div class="Footer-column">
         <div class="Logo Logo--half"></div>
         <p class="Footer-content">
-          Bridging Portugal and Silicon Valley
+          <span>Bridging Portugal</span>
+          <span>and Silicon Valley</span>
```

Comment:
Even though it works, it's a side-effect due to the use of a flexbox.

You want to break the line but `span`s and don't do that since they are, by default, an inline element. The flexbox makes it work but it's semantically incorrect. You can just use a `<br>` instead.